pkg/ovsdb/handler.go

```diff
@@ -107,18 +107,46 @@ func (ch *Handler) Cancel(ctx context.Context, param interface{}) (interface{},
 
 func (ch *Handler) Monitor(ctx context.Context, param ovsjson.CondMonitorParameters) (interface{}, error) {
 	klog.V(5).Infof("Monitor request, parameters %v", param)
-	for tableName, mcr := range param.MonitorCondRequests {
+	ch.mu.Lock()
+	defer ch.mu.Unlock()
+	if ok := ch.createOrDieMonitorsArray(param.JsonValue.(string)); ok != nil {
+		return nil, fmt.Errorf("duplicate json-value")
```

Comment:
why are you not using error returned by createOrDoeMonoitorsArray?

Reply:
fixed